terragrunt/aws/api/lambda.tf

```diff
@@ -1,5 +1,5 @@
 module "url_shortener_lambda" {
-  source                 = "github.com/cds-snc/terraform-modules?ref=v5.0.0//lambda"
+  source                 = "github.com/cds-snc/terraform-modules?ref=v5.0.2//lambda"
```

Comment:
Ah nice catch on the version bump.   It would be nice if we had auto-upgrade PRs for the module versions, but it currently isn't setup.